Task/MCU/Floor1_USART_inputdata/Core/Src/main.c

```diff
@@ -61,13 +64,24 @@ RTC_AlarmTypeDef sAlarm;
 char timeString[MAX_TIME_STRING_LENGTH]; // Time string
 int RTC_Interrupt_flag=0;
 #endif //ifdef R_T_C
+
+#ifdef I_W_D_G
+//IWDG_TIMEOUT value can be between 0 to 32768
+#define IWDG_TIMEOUT 33000
```

Comment:
As you said above comment IWDG_TIMEOUT should be 32768 however you have used 33000 why?
If there is any reason then justify there in comment.

Reply:
Actually, in comment I have mentioned the valid IWDG timeout range, but then also if a user enters the invalid timeout like 33000 ms then it will directly print that desired IWDG timeout is out of range mentioned in comment. I have just run a one of a test case of invalid timeout. Therefore, 33000 is taken.